### DISCUSSION TITLE
openwrt alist插件怎么更新？

### DISCUSSION THREAD

**Discussion by muzhiye** [AlistGo/alist#3518] [Q&A]
设备：N1盒子
OS：Linux OpenWrt 5.15.93-flippy-81+o
alist版本：3.10.1-1
求助，怎么升级到3.11版本？

**Comment by sunburstxm** [comment]
终于找到你，请问N1要装哪个版本的alist

**Reply by muzhiye** [reply]
3.11版本

**Reply by sunburstxm** [reply]
> 3.11版本

啊 不是，我的意思是刷哪个架构版本的

**Reply by muzhiye** [reply]
Built At: 
Go Version: 
Author: 
Commit ID: 
Version: v3.10.1-aarch64
WebVersion: 3.10.1
AArch64是ARMv8 架构的一种执行状态，是这个吗？

**Reply by sunburstxm** [reply]
![Snipaste_2023-02-22_20-27-46](https://user-images.githubusercontent.com/124324799/220619855-62093f99-3c69-43d0-a28f-1f8b69a92b4d.jpg)

**Reply by muzhiye** [reply]
已找到升级方法：
下载alist-linux-musl-arm64.tar.gz 包
解压得到alist，替换openwrt中的/usr/bin/alist
重启服务或在alist点击保存，即可
另外查询软件版本还会是原来版本，但是已经升级了。

**Reply by XingYuyuop** [reply]
谢谢有用

**Comment by zhangxiaojiujiayi** [comment]
alist.com怎么不能登录了?
